### ISSUE TITLE
Script for log bundle collection on VIC OVA

### ISSUE THREAD

**Issue by lgayatri** [vmware/vic-product#1173]
Build: https://storage.googleapis.com/vic-product-ova-releases/vic-v1.3.0-rc2-2628-9e2a95b6.ova

Ask:
 OVA has 14 interlinked components which enables user to execute workflows. Currently in the event of a failure/misbehavior a user files an issue with symptoms and corresponding logs. Often issue may not be at the place where it gets exposed but somewhere in the lower layers. It will be extremely handy if we can provide one script which can collect logs/configurations from all components and bundle into a single tar file. This way a bug being reported will not miss needed information making it easy for root cause analysis. 

This is an enhancement request which will be helpful for all stake holders.
@pdaigle @mdubya66  @martin-borisov @reasonerjt

**Comment by lgayatri** [comment]
These are 14 components I meant
```
vmware/admiral:vic_v1.3.0-rc2
vmware/nginx-photon:1.11.13
vmware/harbor-jobservice:v1.3.0-rc1
vmware/notary-photon:server-0.5.1
vmware/notary-photon:signer-0.5.1
vmware/clair:v2.0.1-photon
vmware/harbor-ui:v1.3.0-rc1
vmware/mariadb-photon:10.2.10
vmware/harbor-adminserver:v1.3.0-rc1
vmware/harbor-db:v1.3.0-rc1
vmware/registry:2.6.2-photon
vmware/postgresql:9.6.5-photon
vmware/harbor-log:v1.3.0-rc1
vmware/vic-machine-server:ova
```

**Comment by jz07** [comment]
@lgayatri : Yes, This is something long awaited, and need of the hour to ensure required information is collected in 1 step for a quick diagnosis and Root cause analysis.

**Comment by andrewtchin** [comment]
Merging with #1197

**Comment by lgayatri** [comment]
@andrewtchin , what is the command to invoke support bundle?
Also , please consider the comment from @morris-jason to include ovfEnv. We recently had a SR in which post OVA restore , the network settings were lost and it was difficult to reconstruct.

**Comment by andrewtchin** [comment]
This issue is closed as a duplicate of #1197. The script to collect the support bundle does not exist yet. That is what #1197 is for.
